analysis/Zr89versusF18.py

```diff
@@ -0,0 +1,200 @@
+import SiemensQuadraProperties as sqp
```

Comment:
what is the purpose of this file?

Reply:
It was used for code development. As it has basically the same functionality as the corresponding jupyter notebook it was moved to a different directory